Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Devices.OpenVR;
+using Microsoft.MixedReality.Toolkit.Internal.Devices.WindowsMixedReality;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
+{
+    /// <summary>
+    /// Helper utility to manage all the required Axis configuration for platforms, where required
+    /// </summary>
+    public static class ControllerMappingLibrary
```

Comment:
This class needs a drastic overhaul it's hard to figure out what axis mapping really goes to what.

Reply:
In beta maybe, it was only temporary anyway and critical for the loading of the InputAxis in the editor.  Might see if I can strip some out now that we've moved to putting the actual mappings in the InteractionMappings.